scripts/main.js

```diff
@@ -268,12 +283,12 @@ function submitImgFormHandler(elem, event) {
       $("#formActionsPostSubmit_"+uid).addClass("d-flex");
       $("#formActionsPostSubmit_"+uid).removeClass("d-none");
       $("#navbarList li[id='link_"+uid+"'] a").html(
-        name + `<span id="deleteBtn_${uid}" class="delete-X"> &nbsp; &times;</span>`
+          makeTabHeaderContent(uid, name)
       );
       $("#deleteBtn_"+uid).click( function(event){deleteImgForm(this);} );
-      
+
       deleteSurvivalGuide(uid, true);
-      
+
```

Comment:
I had a hard time telling my IDE to leave the formatting alone, but failed here. 🤷